docs/docs/reference/contextual/extension-methods.md

```diff
@@ -92,13 +92,13 @@ By contrast, type arguments matching type parameters following `extension` can b
 only if the method is referenced as a regular method:
```

Comment:
It seems the "only if" is not right.
`List[String]("a", "bb", "ccc").sumBy(_.length)` and `List[String]("a", "bb", "ccc").sumBy[Int](_.length)` can work in my Scala 3.0.0-RC1 REPL.

Reply:
Good point. We should add all 4 forms.Can someone make a PR for that?

Reply:
I’m not sure the `List[String]` form can be regarded as a way to pass type parameters to an extension method. We are not passing any parameters to the extension methods, we are helping the compiler to infer the type parameters of the callee precisely. It’s still on the compiler to then infer the type parameters for the extension method; I do not see how explicitly specifying the type parameter of the callee helps the latter task.

Reply:
@odersky Here is the PR: #11451